### PR TITLE
Update tutorial 11

### PR DESCRIPTION
**Pull request by kevin-thankyou-lin**
## Pull request information

- **Status**: Ready to merge
- **Kind of changes**: Bug fix / Documentation
- **Related PR or issue**: ? (optional)

## Description

Updated import modules, fixed typos and removed a version dependent inflow parameter for departure_speed (speedLimit).

### CODE REVIEW

**Comment by eugenevinitsky** [comment]
Question, is it not possible to set a fixed departure speed?

**Comment by kevin-thankyou-lin** [comment]
Oh yes, I can set a fixed departure speed and submit another PR.

**Comment by eugenevinitsky** [comment]
You can also just push another commit to this PR!

File: tutorials/tutorial11_inflows.ipynb [hunk 1]
```diff
@@ -348,7 +334,7 @@
     "- `depart_speed`: this parameter lets you specify the speed at which the vehicles will enter the network. It should be a positive `float`, in meters per second. If this speed is unsafe, the departure of the vehicles is delayed. Just like for `depart_lane`, there are other options for this parameter, which are the following strings:\n",
     "\n",
     "    - `\"random\"`: vehicles enter the edge with a random speed between 0 and the speed limit on the edge. The entering speed may be adapted to ensure that a safe distance to the leading vehicle is kept\n",
-    "    - `\"speedLimit\"`: vehicles enter the edge with the maximum speed that is allowed on this edge. If that speed is unsafe, the departure is delayed.\n",
```
Comment (AboudyKreidieh) [inline tutorials/tutorial11_inflows.ipynb]: don't delete this. replace "speedLimit" with "max", see: https://sumo.dlr.de/docs/Simulation/VehicleInsertion.html

**Comment by kevin-thankyou-lin** [comment]
Got it!

**Comment by kevin-thankyou-lin** [comment]
Made the changes!

**Comment by eugenevinitsky** [comment]
Thanks Kevin!

**Pull request merged**